db/db_local/backup.go

```diff
@@ -271,8 +272,8 @@ func restoreBackup(ctx context.Context) error {
 		utils.ShowWarning("Could not REFRESH Materialized Views while restoring data. Please REFRESH manually.")
 	}
 
-	if err := os.Remove(databaseBackupFilePath()); err != nil {
-		log.Printf("[WARN] Could not remove Backup data at %s.", databaseBackupFilePath())
+	if err := retainBackup(ctx); err != nil {
+		log.Printf("[WARN] Could not retain backup %s.", databaseBackupFilePath())
```

Comment:
utils.ShowWarning?

"Failed to save database backup file"

include error?

Reply:
done

Reply:
so you can remove the log line, right?
```
utils.ShowWarning(fmt.Sprintf("Failed to save backup file %s: %v", databaseBackupFilePath(), err))
```